__tests__/account.test.ts

```diff
@@ -121,7 +114,6 @@ describe('deploy and test Wallet', () => {
       },
     ]);
 
-    expect(code).toBe('TRANSACTION_RECEIVED');
```

Comment:
are there no more transaction status strings?

Reply:
There are none on the RPC side, I can check if they plan to support it

Reply:
please
there must be something like status, how should you otherwise be able to tell if it was rejected/accepted and on which layer L2 or L1 ?

Reply:
You can get the status from the `getTransactionReceipt`